music21/noteworthy/translate.py

```diff
@@ -225,9 +257,7 @@ def setDurationForObject(self, generalNote, durationInfo):
 
         for kk in parts:
             if kk == 'Grace':
-                # Now it doesn't work, the function for grace notes have to be added here
-                warnings.warn('skipping grace note')
-                return
+                durationObject = durationObject.getGraceDuration()
```

Comment:
Thank you for fixing this. I think the warnings filter in `testComplete` can now be removed. Would you be up for cleaning that up?